docs/user_guide/timeseries/forecasting/LagFeatures.rst

```diff
@@ -0,0 +1,257 @@
+.. _lag_features:
+
+.. currentmodule:: feature_engine.timeseries.forecasting
+
+LagFeatures
+===========
+
+The :class:`LagFeatures` adds lag features to the dataframe. A lag feature is a feature
+with information about a prior time step.
+
+In forecasting, past values of the variable we want to forecast, are likely to be
```

Comment:
Suggest change from:

> In forecasting, past values of the variable we want to forecast, are likely to be predictive. 

to

> When forecasting the future values of a time series the past values of that time series are likely to be predictive. 

Reason: Grammatically (I think the comma usage in the first sentence is wrong) correct and clearer imo.